spec/features/metadata_panel_spec.rb

```diff
@@ -44,18 +44,6 @@
       end
     end
     context 'when the metadata is in the XML' do
-      let(:mods) { File.read(Rails.root.join('..', 'spec', 'fixtures', 'mods', 'fb897vt9938.mods')) }
```

Comment:
This block must have been introduced in a rebasing error for #834.  https://github.com/geoblacklight/geoblacklight/pull/834/files#diff-bdf7ae9bd23c14751015614c839f9a31R5 and https://github.com/geoblacklight/geoblacklight/pull/834/files#diff-bdf7ae9bd23c14751015614c839f9a31R15 actually stub the responses with the XML fixtures (in order to avoid transmitting GET requests).